projects/angular/src/wizard/_wizard.clarity.scss

```diff
@@ -558,13 +285,7 @@
 
     .clr-wizard-page:not([aria-hidden='true']) {
       display: block;
-      @include mixins.css-var(
-        padding,
-        clr-wizard-page-padding,
-        variables.$clr_baselineRem_0_75 wizard-variables.$clr-wizard-default-space
-          wizard-variables.$clr-wizard-default-space wizard-variables.$clr-wizard-default-space,
-        variables.$clr-use-custom-properties
-      );
+      padding: 0;
```

Comment:
I think we are not adding padding anywhere so maybe that's not needed.

Reply:
removed in commit b19c78a